plugins/nf-co2footprint/src/resources/assets/CO2FootprintReportTemplate.html

```diff
@@ -282,6 +284,13 @@ <h3 id="measures-plot">Per process</h3>
       <p class="mt-3">The plot shows the distribution of CO₂e emissions and corresponding energy consumptions for each workflow process. The values that are displayed when hovering over the plot refer to the CO<sub>2</sub>e emissions.</p>
     </div>
 
+    <!-- CI records plot -->
+    <div class="container">
+      <h3 id="ci-records">Carbon intensity records</h3>
+      <div id="ci-plot"></div>
+      <p class="mt-3 mb-3">This plot shows the carbon intensity (CI) fluctuation during workflow computation and the energy consumption od individual tasks. Together these two factors form the emitted CO₂e.</p>
```

Comment:
There was a typo and maybe this phrasing sounds a bit clearer

```suggestion
      <p class="mt-3 mb-3">This plot shows the fluctuation of carbon intensity (CI) during workflow computation and the energy consumption of individual tasks. Together, these two factors determine the emitted CO₂e.</p>
```